packages/admin/src/routes/entities/components/EditView/EditView.js

```diff
@@ -40,8 +41,10 @@ const EditView = props => {
 
   const handleNavigateToAction = ({definition, selection}) => {
     const entityBaseUrl = goBack(props.match.url)
+    const searchString = selectionUtil.selectionToQueryString(selection)
```

Comment:
if you would name the var "search" you could use the property shorthand on line 47